python-sdk/docs/conf.py

```diff
@@ -88,7 +88,7 @@
 }
 
 
-def skip_util_classes(app, what, name, obj, skip, options):
+def skip_util_classes(_app, _what, _name, obj, skip, _options):
```

Comment:
and here?

Reply:
???

Reply:
_app, _what, _name, _options unused param so mark them to start with `_`